tests/fixtures/snapshot_date.py

```diff
@@ -21,13 +21,13 @@ def validate_time(timestamp: datetime.datetime | str | Any, _=None):
 
     """
     if isinstance(timestamp, datetime.datetime):
-        if datetime.datetime.utcnow() - timestamp < datetime.timedelta(seconds=30):
+        if datetime.datetime.now() - timestamp < datetime.timedelta(seconds=30):
```

Comment:
I am bit confused about this.

The `datetime.now()` method returns a naive `datetime` object with the local time. The method `utcnow()` returns a naive `datetime` object in UTC time.

The timestamp value should be a naive datetime in UTC, so how is it possible that it is only 30 seconds different from a naive datetime in local time.

I am assuming you read this: https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow

My expectation was that the code about should look like:
```py
if datetime.datetime.now(timezone.utc) - timestamp < datetime.timedelta(seconds=30):
    ...
```
The only explanation I can think of is that `timestamp` is in local time. It shouldn't be.

Reply:
changed from datime.now to arrow.utcnow().naivie to keep it in UTC and Naive